posts/2020-12-15-rust-survey-2020.md

```diff
@@ -0,0 +1,116 @@
+
+![Which versions of Rust do you use?](/images/2020-12-rust-survey-2020/rust-versions.svg)
+
+It’s worth noting that a decent percentage of users who use nightly do so out of habit because “nightly is stable enough”. When asked what broke people’s code most often, by far the largest answer was the introduction of new warnings to a code base where warnings break the build (which is not part of Rust’s stability guarantee though Rust is designed so that adding new warnings never breaks your dependencies). Since we rely on nightly testing to catch regressions, this is a very good sign: nightly is stable enough to be useful while still allowing for continual changes. A shout out to the Rust infrastructure, compiler, and libs teams for doing such a good job of ensuring that what lands in the nightly compiler is already fairly stable!
+
+## Who’s using Rust?
+
+Rust continues to make inroads as a language used for production with roughly 40% of respondents that work in software noting that they use Rust at their day job. Additionally, the future of Rust on the job is bright with nearly half of those who knew saying that their employer planned to hire Rust developers in the next year. 
+
+![Do you use Rust at work?](/images/2020-12-rust-survey-2020/rust-at-work.svg)
+
+The seemingly largest change in those using Rust seems to be students with a much larger percentage (~15% vs ~11% last year) of the respondents answering that they don’t use Rust at work because they’re students or software hobbyists and therefore don’t have a job in software.
+
+Additionally, the use of Rust at respondents' workplaces seems to be getting bigger with 44% of respondents saying that the amount of Rust at work was 10,000 lines of code or more compared to 34% last year. 
+
+![Size of Rust code bases at work](/images/2020-12-rust-survey-2020/project-size.svg)
+
+## Improving Rust
+
+While Rust usage seems to be growing at a healthy pace, the results of the survey made it clear that there is still work to be done to make Rust a more appropriate tool for many people’s workflows.
+
+### C++ Interop
+
+Interestingly, C++ interop was the most requested language for better interop with Rust with C and Python in second and third place. Improved C++ interop was especially often mentioned as a way to improve Rust usage specifically at work. In fact, for users who work on large codebases (100,000 lines of code or larger) C++ interop and unsurprisingly compile times were the most cited ways to improve their Rust experience.    
```

Comment:
This is one of the only two places where compile times are mentioned, and in passing. I know it may feel repetitive, but I would really like to see a small section on compile times, and explicitly talking about how we care about this, and showing the work that's been done. It matters to people a lot.

Reply:
I pushed a change that more explicitly calls out compile times. Please let me know what you think.

Reply:
Looks fantastic, thank you so much.